rootfs/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM quay.io/deis/base:0.2.0
+
+COPY . /
+
+RUN apt-get update \
+	&& apt-get install -y \
+		git \
+		golang \
+		make \
+		--no-install-recommends \
+	&& export DOCKER_REGISTRY_TAG=minio \
+		DOCKER_REGISTRY_REPO=https://github.com/deis/distribution.git \
+		DISTRIBUTION_DIR=/go/src/github.com/docker/distribution \
+	&& export GOPATH=/go:$DISTRIBUTION_DIR/Godeps/_workspace \
```

Comment:
I was just avoiding extra layers since those env vars are not needed later at runtime.